internal/pkg/image_repositories/build_workflow.go

```diff
@@ -7,11 +7,12 @@ import (
 )
 
 var FORKED_PROJECTS map[string]bool = map[string]bool{
-	"horizon":   true,
-	"keystone":  true,
-	"magnum-ui": true,
-	"magnum":    true,
-	"neutron":   true,
+	"horizon":        true,
+	"keystone":       true,
+	"magnum-ui":      true,
+	"magnum":         true,
+	"neutron":        true,
+	"neutron-vpnaas": true,
```

Comment:
i dont think this is gonna work since vpnaas is installed as a pip package right now i believe in the same venv